src/test/eventSchemas/providers/schemasDataProvider.test.ts

```diff
@@ -0,0 +1,108 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
2020, getting tired of adding this so assume it's marked from here on out :P

Reply:
Definitely!